src/lib/components/consignments/__stories__/consignments-bottom-aligned.story.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from "react"
+import { Text, View } from "react-native"
+
+import Search from "../components/artist-search-results"
+import BottomAlignedButton, { BottomAlignedProps } from "../components/bottom-aligned-button"
+
+export const name = "Consignments - bottom aligned"
+export const component = BottomAlignedButton
+
+interface States {
+  [name: string]: BottomAlignedProps
+}
+
+const withText = {
+  children: [<Text style={{color: "white"}}>hi</Text>],
+  onPress: () => "" ,
+}
+
+const withSearchResults = {
+  children: [<Search searching={false} query="Banko" results={null} />],
+  onPress: () => "",
+}
```

Comment:
This is the actual state I wanted to test, as I add more screens, I'll fill this up too 👍